source/agora/node/admin/AdminInterface.d

```diff
@@ -119,6 +120,11 @@ public class AdminInterface : NodeControlAPI
         configureLogger(config, !console.isNull() || !file.isNull());
     }
 
+    // GET: /login
+    public override void login ()
+    {
+    }
```

Comment:
Here we should use the AUTH header to find out if user id and password are correct.

Reply:
We use AUTH header for all the requests. So I introduced the dummy endpoint. It will return an HTTP Status Code(200 or 401). But, I will find a more reasonable way to handle as you mean and the requirement is described.
```
        bool checkPassword (string user, string password) @safe
        {
            return user == config.admin.username && password == config.admin.pwd;
        }
        auto adminrouter = new URLRouter();
        adminrouter.any("*", performBasicAuth("Agora Admin", &checkPassword));
```

Reply:
After thinking more about this we should probably return a JWT which can be used on the subsequent requests to the other endpoints. But for now I think what you have is ok to proceed.